playbook-website/app/javascript/components/Website/src/layouts/Sidebar/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react"
+import { Nav, useCollapsible, Image, Badge, Flex } from "playbook-ui"
+import { TopLevelNavItem } from "./TopLevelNavItems"
+// @ts-ignore
```

Comment:
Why is this needed?

Reply:
It might not be anymore, since I added a custom tsconfig.json for the website. I'll try removing! Thanks dude!